src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -49,6 +49,7 @@ def load_arguments(self, _):
         c.argument('no_wait', options_list=['--no-wait'], arg_group='Timeout', help="Do not wait for the long-running operation to finish.")
         c.argument('correlation_id', options_list=['--correlation-id'], help='A guid that is used to internally track the source of cluster onboarding. Please do not modify it unless advised', validator=override_client_request_id_header)
         c.argument('container_log_path', help='Override the default container log path to enable fluent-bit logging')
+        c.argument('skip_ssl_verification', options_list=['--skip-ssl-verification'], action='store_true', help='Skip SSL verification for any cluster connection.')
```

Comment:
In fact, the default option name does not need to be defined repeatedly
```suggestion
        c.argument('skip_ssl_verification', action='store_true', help='Skip SSL verification for any cluster connection.')
```